apps/web-osmosis/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -176,7 +181,7 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
             </Typography>
           </>
         )}
-        {!!overview.votingEndTime && (
+        {overview?.votingEndTime && (
```

Comment:
I think is safe, just if any of these `overview?.something` is `0` it will display `0` in the UI, but I didn't see any that could be